redis-test/Cargo.toml

```diff
@@ -0,0 +1,18 @@
+[package]
+name = "redis-test"
+version = "0.21.5"
+edition = "2021"
+description = "Testing helpers for the `redis` crate"
+homepage = "https://github.com/redis-rs/redis-rs"
+repository = "https://github.com/redis-rs/redis-rs"
+documentation = "https://docs.rs/redis-test"
+license = "BSD-3-Clause"
+
+[dependencies]
+redis = { path = ".." }
+
+bytes = { version = "1", optional = true }
```

Comment:
Why is `bytes` optional?

Reply:
Because there is a conversion for `bytes::Bytes` in the crate which should only exist if the user is using `bytes::Bytes` in their own crate.